### PR TITLE
interfaces/hardware-observe.go: re-add /run/udev/data

### PR DESCRIPTION
**Pull request by cwayne18**
This was in initially but was incorrectly removed

### CODE REVIEW

**Comment by mvo5** [comment]
:+1:

**Comment by mvo5** [comment]
If @jdstrand is happy this can go in.

File: interfaces/builtin/hardware_observe.go [hunk 1]
```diff
@@ -39,6 +39,9 @@ capability sys_rawio,
 /sys/firmware/dmi/tables/DMI r,
 /sys/firmware/dmi/tables/smbios_entry_point r,
 
+# Needed for udevadm
+/run/udev/data/** r,
```
Comment (jdstrand) [inline interfaces/builtin/hardware_observe.go]: This blanket glob is fine for hardware-observe.

+1

**Comment by mvo5** [comment]
Lets wait for tests, happy to merge once its green

**Pull request merged**